tests/Oryx.BuildImage.Tests/PythonSampleAppsTest.cs

```diff
@@ -859,6 +859,71 @@ public void Build_ExecutesPreAndPostBuildScripts_UsingEnvironmentVariables()
                 result.GetDebugInfo());
         }
 
+        [Fact]
```

Comment:
which version of python does this test target

Reply:
I think we already discussed about this offline.